client/app/scripts/utils/metric-utils.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
+import d3 from 'd3';
+import { formatMetric } from './string-utils';
+import { colors } from './color-utils';
+
+
+const openFilesScale = d3.scale.log().domain([1, 100000]).range([0, 1]);
```

Comment:
I was hoping for these to be more generic. As in, the backed determines the type of scale and the value range. So there would be no need to hardcode `openFiles` here.